source/include/core_mqtt.h

```diff
@@ -58,13 +58,19 @@
 #define MQTT_LIBRARY_VERSION    "v1.2.0"
 /** @endcond */
 
+/**
+ * @ingroup mqtt_constants
+ * @brief Maximum number of vectors in subscribe and unsubscribe packet.
+ */
+#define MQTT_SUB_UNSUB_MAX_VECTORS    ( 4U )
```

Comment:
This seems unused?

Reply:
This is used in the function `sendSubscribeWithoutCopy`.

Reply:
Ahh my diff was missing a file, makes more sense now